postgres-persistence/src/test/java/com/netflix/conductor/postgres/dao/PostgresQueueDAOTest.java

```diff
@@ -71,13 +71,12 @@ public void setup() {
         postgreSQLContainer =
             new PostgreSQLContainer<>(DockerImageName.parse("postgres")).withDatabaseName(name.getMethodName().toLowerCase());
         postgreSQLContainer.start();
-        testUtil = new PostgresDAOTestUtil(postgreSQLContainer, objectMapper, name.getMethodName().toLowerCase());
+        testUtil = new PostgresDAOTestUtil(postgreSQLContainer, objectMapper);
```

Comment:
Is it possible to make `Postgres*DAOTest(s)`  and `MySQL*DAOTest(s)` use `DataJdbcTest`, and rely less on the `TestUtil` classes to create and configure a datasource?

Reply:
Yes, it is possible.
But it will take me another few days.
I think that we can merge this PR without it, and then I will open a new PR  (this issue prevents from the conductor to startup).

Reply:
Sounds good.